.teamcity/Gradle_Check/configurations/IndividualPerformanceScenarioWorkers.kt

```diff
@@ -14,9 +14,9 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.AbsoluteId
 import model.CIBuildModel
 
 class IndividualPerformanceScenarioWorkers(model: CIBuildModel, os: Os = Os.LINUX) : BaseGradleBuildType(model, init = {
-    uuid = model.projectPrefix + "IndividualPerformanceScenarioWorkers${os.name.toLowerCase().capitalize()}"
+    uuid = model.projectPrefix + "IndividualPerformanceScenarioWorkers$os"
```

Comment:
I am not convinced we should use the `toString()` representation for the ID, even though it is the same string now.

Reply:
How about `asName` or `asTypeId`?